shared/js/shared-utils/parse-user-agent-string.es6.js

```diff
@@ -33,6 +33,7 @@ module.exports = (uaString) => {
     return {
         os,
         browser,
-        version
+        version,
+        manifestVersion: chrome.runtime.getManifest().manifest_version
```

Comment:
Is there a reason to export this over using the `browserWrapper.getManifestVersion()`?

Reply:
No - I've updated the code to use the wrapper instead.